frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -247,6 +255,144 @@ const ChannelHeader: React.FC<{}> = () => {
+      className={classNames('co-channel-version-dot', {
+        'co-channel-version-dot--current': current,
+      })}
+    ></div>
+  );
+};
+
+const getChannelPrefix = (channel: string): string => channel.split('-')[0];
+const getChannelVersion = (channel: string): number => parseFloat(channel.split('-')[1]);
+
+const UpdatesGraph: React.FC<UpdatesGraphProps> = ({ cv }) => {
+  const status = getClusterUpdateStatus(cv);
+  const availableUpdates = getSortedUpdates(cv);
+  const upToDate = status === ClusterUpdateStatus.UpToDate;
+  const desiredVersion = getDesiredClusterVersion(cv);
+  const lastVersion = getLastCompletedUpdate(cv);
+  const newestVersion = availableUpdates[0]?.version;
+  const secondNewestVersion = availableUpdates[1]?.version;
+  const currentChannel = cv.spec.channel;
+  const similarChannels = _.keys(getAvailableClusterChannels()).filter((channel) =>
+    channel.includes(getChannelPrefix(currentChannel)),
+  );
+  const newerChannel = similarChannels.find(
+    // find the next minor version, which should never be more than one
+    (channel) => _.round(getChannelVersion(channel) - getChannelVersion(currentChannel), 1) === 0.1,
+  );
+
+  return (
+    status !== ClusterUpdateStatus.ErrorRetrieving && (
+      <div className="co-cluster-settings__updates-graph">
+        <Channel upToDate={upToDate}>
+          <ChannelPath current={true}>
+            <ChannelLine>
+              <ChannelVersion current={true}>{lastVersion}</ChannelVersion>
+              <ChannelVersionDot current={true} />
+            </ChannelLine>
+            {upToDate && <ChannelLine />}
+            {availableUpdates.length === 2 && (
+              <ChannelLine>
+                <ChannelVersion>{secondNewestVersion}</ChannelVersion>
+                <ChannelVersionDot />
+              </ChannelLine>
+            )}
+            {availableUpdates.length > 2 && (
+              <ChannelLine>
+                <Button
+                  variant="secondary"
+                  className="co-channel-more-versions"
+                  onClick={() => clusterMoreUpdatesModal({ cv })}
+                >
+                  + More
+                </Button>
+              </ChannelLine>
+            )}
+            {(newestVersion || status === ClusterUpdateStatus.Updating) && (
+              <ChannelLine>
+                <ChannelVersion>{newestVersion || desiredVersion}</ChannelVersion>
+                <ChannelVersionDot />
+              </ChannelLine>
+            )}
+          </ChannelPath>
+          <ChannelName current={true}>{currentChannel}.channel</ChannelName>
+        </Channel>
+        {newerChannel && (
+          <Channel>
+            <ChannelPath>
+              <ChannelLine start={true}>
+                <div className="co-channel-switch"></div>
+              </ChannelLine>
+              <ChannelLine />
+              <ChannelLine />
+            </ChannelPath>
+            <ChannelName>{newerChannel}.channel</ChannelName>
```

Comment:
Did you mean

```suggestion
            <ChannelName>{newerChannel.channel}</ChannelName>
```

Reply:
Whoops.  That should be `<ChannelName>{newerChannel} channel</ChannelName>`